src/commands/init.rs

```diff
@@ -91,55 +86,3 @@ fn write_config(mut config: File) {
         panic!("could not write to config file, because: {why}");
     }
 }
-
-/// Creates a config file at the location
-/// - **Linux:** `$HOME/.config/subspace-cli/settings.toml`.
-/// - **macOS:** `$HOME/Library/Application Support/subspace-cli/settings.toml`.
-/// - **Windows:** `{FOLDERID_RoamingAppData}/subspace-cli/settings.toml`.
-pub(crate) fn create_config() -> (File, PathBuf) {
-    let config_path = match dirs::config_dir() {
-        Some(path) => path,
-        None => panic!("couldn't get the default config directory!"),
-    };
-    let config_path = config_path.join("subspace-cli");
-    let _ = create_dir(config_path.clone()); // if folder already exists, ignore the error
-
-    match File::create(config_path.join("settings.toml")) {
-        Err(why) => panic!("couldn't create the config file because: {}", why),
-        Ok(file) => (file, config_path),
-    }
-}
-
-fn construct_config(
-    reward_address: &str,
-    plot_location: &str,
-    plot_size: &str,
-    chain: &str,
-    node_name: &str,
-) -> String {
-    format!(
-        "[farmer]
-address = \"{}\"
-sector_directory = \"{}\"
-sector_size = \"{}\"
-opencl = false
-
-[node]
-chain = \"{}\"
-execution = \"wasm\"
-blocks-pruning = 1024
-state-pruning = 1024
-validator = true
-name = \"{}\"
-port = 30333
-unsafe-ws-external = true # not sure we need this
-
-[chains]
-gemini-1 = \"rpc://1212312\"
-gemini-2= \"rpc://\"
-leo-3 = \"myown-network\"
-dev = \"that local node experience\"
-",
```

Comment:
I think it might be cleaner to just serialize a struct into [toml](https://docs.rs/toml/latest/toml/) and output this `to_string`?